plugins/frontend/backstage-plugin-aws/README.md

```diff
@@ -0,0 +1,13 @@
+# aws-frontend
+
+Welcome to the aws-frontend plugin!
```

Comment:
Could you pls explain here what are these cards? How can someone sset this plugin up? Every plugin can be a first interaction for a potential Backstage user so we should explain what's in this package and how can they add the cards to their instances

Reply:
done.